qa/rpc-tests/wallet_z_sendmany.py

```diff
@@ -181,7 +208,32 @@ def run_test(self):
         # Send some funds to a specific legacy taddr that we can spend from
         recipients = []
         recipients.append({"address":mytaddr, "amount":5})
-        opid = self.nodes[0].z_sendmany(n0ua0, recipients, 1, 0)
+
+        # If we attempt to spend with the default privacy policy, z_sendmany
+        # returns an error because it needs to create a transparent recipient in
+        # a transaction involving a Unified Address.
+        assert_raises_message(
+            JSONRPCException,
+            'AllowRevealedRecipients',
+            self.nodes[0].z_sendmany,
+            n0ua0, recipients, 1, 0)
+
+        # If we set any policy that does not include AllowRevealedRecipients,
+        # z_sendmany also returns an error.
+        for policy in [
+            'FullPrivacy',
+            'AllowRevealedAmounts',
+            'AllowRevealedSenders',
+            'AllowLinkingAccountAddresses',
+        ]:
+            assert_raises_message(
+                JSONRPCException,
+                'AllowRevealedRecipients',
+                self.nodes[0].z_sendmany,
+                n0ua0, recipients, 1, 0, policy)
+
+        # By setting the correct policy, we can create the transaction.
```

Comment:
Again, test that seeing a weaker policy works.

(No need to re-test using an unrecognised name.)